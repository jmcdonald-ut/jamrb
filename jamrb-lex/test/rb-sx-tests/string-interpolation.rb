"#{1 + 1} should be 2"
'#{1 + 1} should be a string.'
