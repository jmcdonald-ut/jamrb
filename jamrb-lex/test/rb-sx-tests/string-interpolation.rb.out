((Token ((1 0) on_tstring_beg "\\\""))
 (Token ((1 1) on_embexpr_beg "\\#{"))
 (Token ((1 3) on_int "1"))
 (Token ((1 4) on_sp " "))
 (Token ((1 5) on_op "+"))
 (Token ((1 6) on_sp " "))
 (Token ((1 7) on_int "1"))
 (Token ((1 8) on_embexpr_end "}"))
 (Token ((1 9) on_tstring_content " should be 2"))
 (Token ((1 21) on_tstring_end "\\\""))
 (Token ((1 22) on_nl "\\n"))
 (Token ((2 0) on_tstring_beg "'"))
 (Token ((2 1) on_tstring_content "\\#{1 + 1} should be a string."))
 (Token ((2 29) on_tstring_end "'"))
 (Token ((2 30) on_nl "\\n")))
